modules/olm-installing-specific-version-cli.adoc

```diff
@@ -30,9 +30,131 @@ endif::[]
 
 .Procedure
 
+. Look up the available versions and channels of the Operator you want to install by running the following command:
++
+.Command syntax
+[source,terminal]
+----
+$ oc describe packagemanifests <operator_name> -n <catalog_namespace>
```

Comment:
I guess we can describe this command for the users: `$ oc get packagemanifests <operaror_name> -n <catalog_namespace> -ojsonpath='{.status.channels[?(@.name=="<channel_name>")].entries}'|jq .` such as,
```yaml
MacBook-Pro:~ jianzhang$ oc get  packagemanifests quay-operator -n openshift-marketplace -ojsonpath='{.status.channels[?(@.name=="stable-3.8")].entries}'|jq .
[
  {
    "name": "quay-operator.v3.8.5",
    "version": "3.8.5"
  },
  {
    "name": "quay-operator.v3.8.4",
    "version": "3.8.4"
  },
  {
    "name": "quay-operator.v3.8.3",
    "version": "3.8.3"
  },
  {
    "name": "quay-operator.v3.8.2",
    "version": "3.8.2"
  },
  {
    "name": "quay-operator.v3.8.1",
    "version": "3.8.1"
  },
  {
    "name": "quay-operator.v3.8.0",
    "version": "3.8.0"
  }
]
MacBook-Pro:~ jianzhang$ oc get  packagemanifests quay-operator -n openshift-marketplace -ojsonpath='{.status.channels[?(@.name=="stable-3.7")].entries}'|jq .
[
  {
    "name": "quay-operator.v3.7.11",
    "version": "3.7.11"
  },
  {
    "name": "quay-operator.v3.7.10",
    "version": "3.7.10"
  },
  {
    "name": "quay-operator.v3.7.9",
    "version": "3.7.9"
  },
  {
    "name": "quay-operator.v3.7.8",
    "version": "3.7.8"
  },
  {
    "name": "quay-operator.v3.7.7",
    "version": "3.7.7"
  },
  {
    "name": "quay-operator.v3.7.6",
    "version": "3.7.6"
  },
  {
    "name": "quay-operator.v3.7.5",
    "version": "3.7.5"
  },
  {
    "name": "quay-operator.v3.7.4",
    "version": "3.7.4"
  },
  {
    "name": "quay-operator.v3.7.3",
    "version": "3.7.3"
  },
  {
    "name": "quay-operator.v3.7.2",
    "version": "3.7.2"
  },
  {
    "name": "quay-operator.v3.7.1",
    "version": "3.7.1"
  },
  {
    "name": "quay-operator.v3.7.0",
    "version": "3.7.0"
  }
]
```

Reply:
Thanks for the feedback, @jianzhangbjz. 

In the documentation, we try to avoid using `jq` when other options are available. This rule comes from support and PX so customers don't have to install `jq`. I guess installing `jq` is a problem in some environments.